Individual.cpp

```diff
@@ -216,5 +217,31 @@ ostream& operator<<(ostream &fout, Individual& ind)
 	return fout;
 }
 
+void Individual::setIndividualMatchFile()
+{
+	string ext = ".tsv";
+	string dir = "individual_match_files/";
+	string fileHandleName = dir + single_id + ext;
+	individualMatchFile = new ofstream(fileHandleName, ofstream::app);
+}
+
+void Individual::setIndividualHomozFile()
+{
+	string ext = ".tsv";
+	string dir = "individual_homoz_files/";
```

Comment:
Folder name should be `dog_level_homoz_files` to preserve behaviour of existing rust code?